### ISSUE TITLE
Is it possible to get this to work on chrome-extension:// pages?

### ISSUE THREAD

**Issue by narcolepticinsomniac** [tophf/chromeContextDelete#1]
I'm not gonna be surprised if the answer is "no", but I had to ask. Maybe there's a hack.

**Comment by tophf** [comment]
You can enable `chrome://flags/#extensions-on-chrome-urls` flag.
Chrome will display an inline warning on every start though.

**Comment by narcolepticinsomniac** [comment]
Nah, I tried that before. Unless I'm missing something, it still doesn't work. That inline warning is annoying af too, so even if it did, I'm not sure it'd be an even trade-off. Any chance we could add this to the Stylish fork? It'd be nice if it worked for all Chrome extensions, but Stylish is where I really miss it.
